WL-3743 Fixed the 'WorksiteSetup' error caused by user search in site…

ouit0408 opened the pull request:
…-info.

Used new variable 'SITE_USER_SEARCH' for storing user search term
in SiteAction class. On 'Prev' and 'Next' button click the search attribute
is removed from the state.

buckett commented on site-manage-tool/tool/src/java/org/sakaiproject/site/tool/SiteAction.java:
```diff
@@ -665,6 +665,8 @@ public SecurityAdvice isAllowed(String userId, String function, String reference
 
 	private static final String STATE_SITE_PARTICIPANT_LIST = "site_participants";
 
```
Could you just JavaDoc this variable, so it's really clear what it's used for.

ouit0408 committed:
WL-3743 Fixed the 'WorksiteSetup' error caused by user search in site-info.

Used new variable 'SITE_USER_SEARCH' for storing user search term
in SiteAction class. On 'Prev' and 'Next' button click the search attribute
is removed from the state.

ouit0408 commented:
Made the change.
